Automattic/woocommerce-payments
# Update subscriptions-core to 6.2

Subscriptions core 6.2.0 full changelog:

```
* Add - Introduce an updated empty state screen for the WooCommerce > Subscriptions list table.
* Fix - Ensure subscription checkout and cart block integrations are loaded on store environments where WooPayments is not enabled.
* Fix - Ensure the shipping phone number field is copied to subscriptions and their orders when copying address meta.
* Update - When HPOS is disabled, fetch subscriptions by customer_id using the user's subscription cache to improve performance.
* Dev - Deprecated the 'woocommerce_subscriptions_not_found_label' filter.
```

Note when copying these entries into WooPayments, I've excluded the empty state screen (1) and checkout block entries (2). Both of these are not relevant to WooPayments because:

1. The empty state screen added to subscriptions-core was already in WooPayments. There's no change. 
2. The checkout block compatibility fix only applied on environments where WooPayments wasn't enabled. 

### Testing instructions
1. Checkout the branch
2. Run composer install
3. Confirm the latest subscriptions core version is found in vendor/woocommerce/subscriptions-core
